jsapp/js/utils.es6

```diff
@@ -115,7 +115,8 @@ export function unnullifyTranslations(surveyDataJSON, assetContent) {
             if (typeof surveyData.settings[0] !== 'undefined'
                 && typeof surveyData.settings[0].style === 'string'
                 && surveyData.settings[0].style.includes('theme-grid')
-                && surveyRow.type === 'begin_group') {
+                && surveyRow.type === 'begin_group'
+                && surveyRow[translatedProp] === null || surveyRow[translatedProp] === '') {
```

Comment:
The `&&` operator has higher precedence than `||`, so I don't think this does exactly what you want. If `surveyRow[translatedProp] === ''` is `true`, than the whole expression evaluates to `true`, regardless of everything that comes before the logical or operator. As a simplified example:
```
Welcome to Node.js v12.18.3.
Type ".help" for more information.
> false && false && false || true
true
```
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence#Table.

(All of this is a long-winded way to say "add a few parentheses" :wink:)

Reply:
Checked just in case my tests were only working because ` || surveyRow[translatedProp] === ''` was taking precedence, added parentheses and it still works :+1: